src/refresh.js

```diff
@@ -0,0 +1,14 @@
+import Result from './result.js';
+
+export default async function refresh() {
```

Comment:
- [ ] also here please try to use the arrow function again for more detail check the comment above